recipe/meta.yaml

```diff
@@ -23,15 +22,13 @@ source:
     - 0001-shobjidl-Fix-compile-guard-around-SHARDAPPIDINFOLINK.patch  # [win]
 
 build:
-  number: 2  # [win]
-  number: 4  # [linux]
-  number: 5  # [osx]
```

Comment:
Multiple build number are elegant but error prone. In conda-forge we use a single one even if the other platforms did not change.